src/applications/personalization/dashboard/components/health-care-v2/HealthCareV2.jsx

```diff
@@ -130,6 +130,8 @@ HealthCareV2.propTypes = {
   unreadMessagesCount: PropTypes.number,
 };
 
+export const UnconnectedHealthCareV2 = HealthCareV2;
+
 export default connect(
```

Comment:
The [`connect()` documentation](https://react-redux.js.org/api/connect) says we should use the [hooks API](https://react-redux.js.org/api/hooks) by default, but maybe this may not work for functional components

Reply:
I think the idea is that you can unit test a 'redux connected' component in 2 different ways. Render the component inside a provided state store, or render it unconnected from the state store and pass all the state that would normally come from redux as plain ole props.

Reply:
I'm okay with the way it is, but it might warrant a discussion on testing best practices and if we should mock the state store or use an Unconnected component and pass state directly as props.

It can be a giant pain to wire up a redux store for a test, especially since ours is gigantic, but then if you do, you know you are mapping the right state to props, so im kinda 🤷🏼‍♂️ on how that should be done.

The alternative that @allisonlu may be getting at, is some thing like this I think:

```
const tree = renderWithStoreAndRouter(<HealthCareV2 />, {
      initialState: {
        all the initial state here so the component can render
      },
      reducers: {
        the My Va state reducers
      }
    });
    ```